operator/src/lib.rs

```diff
@@ -0,0 +1,86 @@
+mod operator;
+
+use std::collections::HashMap;
+use std::ffi::{OsStr, OsString};
+use std::path::{Path, PathBuf};
+use std::sync::Arc;
+
+use anyhow::bail;
+use clap::Args;
+use reth_primitives::Address;
+
+pub use operator::Operator;
+
+#[derive(Args)]
+pub struct Config {
```

Comment:
kinda weird to add cli args to libs don't you think?

Reply:
not as weird as having a 20+ field struct in `proxy` crate with no structure whatsoever IMO. 

I believe that defining a config layout is a responsibility of a lib.